src/Event/Productions/SimilarEventPair.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+class SimilarEventPair
+{
+    /**
+     * @var string
+     */
+    private $eventOne;
+
+    /**
+     * @var string
+     */
+    private $eventTwo;
+
+    public function __construct(string $eventOne, string $eventTwo)
+    {
+        $this->eventOne = $eventOne;
+        $this->eventTwo = $eventTwo;
+    }
+
+    public static function fromArray(array $eventIds) : SimilarEventPair
+    {
+        if (!array_key_exists(0, $eventIds) || !array_key_exists(1, $eventIds)) {
+            throw new \InvalidArgumentException();
+        }
+
+        return new self($eventIds[0], $eventIds[1]);
+    }
+
+    public function asArray(): array
```

Comment:
I think we can remove this method as it's only used in a unit test. We would need to adapt the unit test maybe to verify deserialization?

Reply:
I'll remove the method in the follow-up refactor